azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -1591,7 +1603,7 @@ private String getVPAName(final int recommendationId) {
    * @return
    */
   private String getServiceName(final int executionId) {
```

Comment:
Shall we create a static function to getServiceName and let getFlowServiceEndpoint to consume that static function.

Given we have two similar impls about getting the service name (one static; one non-static).

Reply:
The non-static one is calling the static one to get the service name - we only keep the actual implementation in the static method.